src/main/java/net/coderbot/iris/uniforms/CommonUniforms.java

```diff
@@ -59,6 +70,27 @@ private static float getBlindness() {
 		return 0.0F;
 	}
 
+	private static float getPlayerMood() {
+		if (client.player == null) {
+			return 0.0F;
+		}
+		return client.player.getMoodPercentage();
```

Comment:
Blank lines between braces and statements
```suggestion

		return client.player.getMoodPercentage();
```

Reply:
I've added a new line but I don't think git has detected it, so I will just resolve this issue.